homeassistant/components/tmdb/sensor.py

```diff
@@ -0,0 +1,123 @@
+"""Support for The Movie Database API."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY, ATTR_ID, ATTR_ATTRIBUTION)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['tmdbsimple==2.2.0']
+
+DOMAIN = 'tmdb'
```

Comment:
Please remove this.

Reply:
Why? It's used in my test as well

Reply:
Platforms shouldn't define the domain constant. Components do that.